cmd/main.go

```diff
@@ -207,15 +207,9 @@ func main() {
 		os.Exit(1)
 	}
 
-	restClient, err := newRestClient(cfg)
+	dynamicClient, err := newDynamicClient(cfg)
```

Comment:
I guess that these changes should be also compatible with other API like AAP (we use the dynamic client also there)

Reply:
ok, fair enough. Thanks for explanation :+1: